frontend/views/components/Transitions/MaskToModal/MaskToModal.vue

```diff
@@ -1,8 +1,8 @@
-<template>
-  <component :is="tag">
-    <slot></slot>
-  </component>
+<template lang="pug">
+  component(:is="tag")
```

Comment:
I have not seen elsewhere where you use double-quotes for the value of an attribute... so why are you using it here?